### DISCUSSION TITLE
unable to find main-class

### DISCUSSION THREAD

**Discussion by Saitejakst** [kokorin/Jaffree#349] [Q&A] [answered]
I need ffmpeg.jar as an runnable jar file for my NiFi application.

I have generated jaffree-ffmpeg.jar using Maven Clean package.
while i was using cmd line "java -jar jaffree-ffmpeg.jar"
i have got an error like 
"Error: Could not find or load main class com.example.Main
Caused by: java.lang.ClassNotFoundException: com.example.Main"

can you help me or if there is any alternative way to get the runnable ffmpeg jar file for NiFi could you please let me know?

**Comment by kokorin** [comment, marked as answer]
But why do you ask such question here? The error can be checked on stack
overflow.
If you really want an answer from  me: are you sure such file with Main
class is in jar? Show directory tree in jar file.

чт, 20 апр. 2023 г., 12:31 sai teja ***@***.***>:
